contracts/access/OwnedClaimable.sol

```diff
@@ -33,19 +35,20 @@ abstract contract OwnedClaimable {
                   CONSTRUCTOR
     //////////////////////////////////////*/
 
-    constructor() {
-        owner = msg.sender;
+    constructor(address _owner) {
```

Comment:
Inside the constructor, I opted to use the `FilAddress` lib straight up, rather than `using FilAddress for address` for the entire contract. My assumption here is that - because normalization would only happen in the constructor - bringing in the full library adds extra bloat that should otehrwise get discarded after deployment

I'd need some more time to actually test this assumption... So happy to use the standard `using FilAddres for address` to keep consistent if you prefer

Reply:
You don't bring the whole library into the contract with a `using...for` declaration. Solc only adds the individual functions you call.